Lib/_pyrepl/__main__.py

```diff
@@ -1,7 +1,11 @@
 import os
 import sys
 
-CAN_USE_PYREPL = sys.platform != "win32"
+CAN_USE_PYREPL: bool
+if sys.platform != "win32":
+    CAN_USE_PYREPL = True
+else:
+    CAN_USE_PYREPL = sys.getwindowsversion().build >= 10586  # Windows 10 TH2
```

Comment:
This merged as-is which is OK, but I wanted to chip in...

There is a _minimal_ risk of cutting out terminals that would support pyrepl on earlier versions of Windows. The "most correct" (also most soapboxy, sorry!) way to determine support for this is to just try to set `ENABLE_VIRTUAL_TERMINAL_PROCESSING` and only fall back to basic console APIs if it fails. :)

Reply:
@DHowett sounds like an improvement we can still make! Care to PR this?

Reply:
I can try - at the very least, it would be a great opportunity for me to learn how to build and deploy Python 🙂

Reply:
@DHowett see here and let me know if anything's missing:
https://devguide.python.org/getting-started/setup-building/#windows